Proposal - Adjust the BAT-A Debt Ceiling - Sepember 17, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Filename: September 21 (date the poll goes live.) Also lose 'proposal' in front, I think we're going to stop using that (it's been really inconsistent in the past anyway.)